Sources/SwiftCrossUI/Views/ProgressView.swift

```diff
@@ -114,8 +126,33 @@ struct ProgressSpinnerView: ElementaryView {
         backend: Backend,
         dryRun: Bool
     ) -> ViewUpdateResult {
-        ViewUpdateResult.leafView(
-            size: ViewSize(fixedSize: backend.naturalSize(of: widget))
+        let naturalSize = backend.naturalSize(of: widget)
+        guard isResizable else {
+            // Required to reset its size when resizability
+            // gets changed at runtime
+            backend.setSize(of: widget, to: naturalSize)
+            return ViewUpdateResult.leafView(size: ViewSize(fixedSize: naturalSize))
+        }
+        let min = max(min(proposedSize.x, proposedSize.y), 10)
```

Comment:
Do we want to have a lower bound of 10? I feel like we should just let the spinner vanish to 0 by 0 if sufficiently squished (like we do for other resizable views such as images or shapes). 10 is also backend-specific, so may not be a suitable minimum for all backends.